src/geo/leaflet/leaflet-layer-view-factory.js

```diff
@@ -0,0 +1,49 @@
+var log = require('cdb.log');
```

Comment:
@viddo Here's an example of a factory that I'm introducing here. Some other factories that we have in CartoDB.js provide a way to register new types and constructors but I've decided to go with this very simple alternative (no new types will need to be added at runtime). I'd like to hear you thoughts on this! Thx